db/db_universal_compaction_test.cc

```diff
@@ -1680,6 +1680,7 @@ TEST_P(DBTestUniversalCompaction, ConcurrentBottomPriLowPriCompactions) {
   Env::Default()->SetBackgroundThreads(1, Env::Priority::BOTTOM);
   Options options = CurrentOptions();
   options.compaction_style = kCompactionStyleUniversal;
+  options.max_background_compactions = 2;
```

Comment:
`max_background_compactions` is not supported according to our doc. Should we switch to max_background_jobs?

Reply:
Changed the doc to promote `max_background_compactions` and `max_background_flushes` from unsupported to deprecated. We don't have a path to get rid of them yet.